src/main/daml/Daml/Finance/Interface/Holding/Transferable.daml

```diff
@@ -44,14 +44,16 @@ interface Transferable where
     do
       pure $ view this
 
-  choice Transfer : ContractId Transferable
+  nonconsuming choice Transfer : ContractId Transferable
```

Comment:
Is there a reason this _has to be_ non-consuming, ie. are there situations where the holding is not consumed? It's a bit counterintuitive, and wouldn't we always want to consume it anyway?

Reply:
ideally, a post-consuming choice would be more intuitive but that is currently not supported in interfaces. the reason it can't be immediately consumed is that it is being consumed in the `debit` choice.